atlp-rwanda/Team-furebo-E-commerce-bn
# #184651324 App Logout Functionality

### **What does this PR do?**
Adds log-out functionality to the project
##### **Description of Task to be completed?**
Users should be able to log out of the application
As a user, I want to click on the "logout" link So 
that I get logged out of the application When a 
user clicks on the logout link, they should be 
signed out of the application with their session ended.

#### How should this be manually tested?
- First Clone the repository  
- Navigate to the branch project 
- use **`npm install`** to install the required dependencies.
- to test use **`npm run test`** and see the coverage.
- use **`npm start`** to start the server
- Navigate to the project URL `http://localhost:3000`
- Login with email and you will be redirected to 
  the protected page
- Click the **Logout** button to log out.

#### What are the relevant pivotal tracker/Trello stories?
[#184651324](https://www.pivotaltracker.com/n/projects/2633635)
